orbit/pkg/execuser/execuser_windows.go

```diff
@@ -105,6 +107,10 @@ const (
 //	create an updated block for the child process to inherit, create the child process, and then
 //	restore the saved values using SetEnvironmentVariable, as shown in the following example."
 func run(path string, opts eopts) error {
+	if err := setupSyncChannel(constant.DesktopAppExecName); err != nil {
```

Comment:
No need to tackle this now, but the `channelID` could be passed via `eopts` (reasoning the `execuser` package doesn't really need/have Fleet Desktop stuff hardcoded).

Reply:
that's a good point. I kind of abused the constant mainly to not have to keep the `channelID` value and be able to call SignalProcessBeforeTerminate() from random places but is actually not needed at this point. So this is a good improvement to make in the future.